### DISCUSSION TITLE
can't generate agent installation .exe

### DISCUSSION THREAD

**Discussion by DreNess7755** [amidaware/tacticalrmm#1536] [General]
**Installation Method:**
  - [ x] Standard
  - [ ] Docker

**Describe the bug**

Impossible to generate an installation file in .exe from the rmm dashboard ( powershell and manual works fine). Error 400 bad request.

I remember actualy getting it to work but in local couple months ago.

**Screenshots**

![msedge_5VQkQyUBFp](https://github.com/amidaware/tacticalrmm/assets/40301239/d0bd65fa-ccc5-42e7-b8e5-4198b3c4d06a)

![msedge_So7BROvFw0](https://github.com/amidaware/tacticalrmm/assets/40301239/755a6472-df26-4a7a-b4b7-7d6f229927ea)

![msedge_G0WJemiodG](https://github.com/amidaware/tacticalrmm/assets/40301239/83e39ad4-9ed8-4ab7-bfb0-b842716a2362)

**Comment by silversword411** [comment]
Can you ping `agents.tacticalrmm.com`? Looks like your DNS is broken

**Reply by DreNess7755** [reply]
No I can't ping this name particularly, what DNS issu that could be ?
I have my 3 names configured like 
api.exemple.com
mesh.exemple.com
rmm.exemple.com

**Reply by DreNess7755** [reply]
Ok that was a borken DNS like you said in my network configuration... i feel stupid thanks for the tip !

**Comment by wh1te909** [comment]
also if you have outbound firewall, you need to allow communication to `agents.tacticalrmm.com` to port 443

**Reply by DreNess7755** [reply]
I have disabled firewall on Debian and on my ISP router but still does not work

**Reply by DreNess7755** [reply]
Ok that was a borken DNS like said before in my network configuration... i feel stupid thanks for the tip !
